dev/all_tests.yaml

```diff
@@ -218,6 +218,21 @@ tests:
       test_gcs_bucket: $storage_gcs_bucket
       branch: master
 
+  gcs_pubsub_gae_test:
+    requires:
+      configuration:
+        appengine_account_enabled: true
+      services: [clouddriver, front50, gate, orca, echo]
+    quota:
```

Comment:
i think you need to account for the LBs here too.
Note that this is only used from the validation controller scripts, not the test itself, so you might want the test to verify the quota is available if this is sensitive.

Reply:
I don't know what quota needs accounted for, I'll add a TODO and ask Dan.

Reply:
The project has quota. The test controller is allocated a certain amount of quota to work with and manages that quota at a test script level using this config. So if the test is creating load balancers, and there are project quotas, then this test needs to specify how many to allocate it when managing script concurrency.

Reply:
Is pub sub always available or does it require explicit configuration? If it is opt in then you should have something in the configuration section that tests for it so that the test will be skipped if the deployment configuration is not applicable. Your test itself should test for it if it can.

This config file is tied to the validate_bom script and the parameters here in the configuration are the configuration parameters to the deploy step. The test deployed the servers so it knows how they were configured. This file ties the deployment configuration parameters into the testing configuration. So you can test for the configuration here to specify what's needed.